deploy/sample-backup-configs/sample_config_build_farm.yaml

```diff
@@ -0,0 +1,58 @@
+### Build Farm Config File ###
+
+###########
+# Schema:
+###########
+# # Unique name for build farm. Build farm named "default-build-farm" will
+# # be the default build farm used for all recipes without a specific build
+# # farm used.
+# <NAME>:
+#     # Human readable name of the build farm type.
+#     # This can be determined from the `NAME` field in `BuildFarmHostDispatcher`
+#     # child class (located in `deploy/buildfarmhostdispatcher.py`).
+#     build-farm-type: <TYPE NAME>
+#     args:
+#         # Build farm arguments that are passed to the `BuildFarmHostDispatcher`
+#         # object. Determined by looking at `parse_args` function of class.
+#         <K/V pairs of args>
+
+# THIS WILL BE USED IF BUILDFARM NOT SPECIFIED IN RECIPE
+# Note: For large designs (ones that would fill a EC2.2xlarge/Xilinx VU9P)
+# Vivado uses in excess of 32 GiB. Keep this in mind when selecting a
+# non-default instance type.
+default-build-farm:
+    build-farm-type: aws-ec2
+    args:
+        # REQUIRED: instance type to use per build
+        instance-type: z1d.2xlarge
+        # REQUIRED: instance market to use per build
+        build-instance-market: ondemand
+        # REQUIRED: if using spot instances, determine the interrupt behavior
+        spot-interruption-behavior: terminate
+        # REQUIRED: if using spot instances, determine the max price
+        spot-max-price: ondemand
+        # OPTIONAL: change location of build directory on build host.
```

Comment:
Wait... for both of these becoming mandatory, would we allow for environment variables to be passed into it? Or would we force it to be an absolute path? If its an absolute path then this would have to be mainly for EC2 instances only (`/home/centos`)